.palantir/revapi.yml

```diff
@@ -1,6 +1,17 @@
 versionOverrides:
   org.apache.iceberg:iceberg-api:apache-iceberg-0.14.0: "0.14.0"
 acceptedBreaks:
+  list:
+    org.apache.iceberg:iceberg-api:
+    - code: "java.field.removed"
+      old: "field org.apache.iceberg.ManifestFile.ADDED_FILES_COUNT"
+      justification: "Removes inconsistency between code and spec"
+    - code: "java.field.removed"
+      old: "field org.apache.iceberg.ManifestFile.DELETED_FILES_COUNT"
+      justification: "Removes inconsistency between code and spec"
+    - code: "java.field.removed"
+      old: "field org.apache.iceberg.ManifestFile.EXISTING_FILES_COUNT"
+      justification: "Removes inconsistency between code and spec"
```

Comment:
Since we are going to be releasing 1.0 and can't deprecate these properly, can we leave the static field name intact and make it an alias for the correct one?

Reply:
I like that a lot, thanks 👍🏻